backend/mediaprovider/subsonic/artistiterator.go

```diff
@@ -46,6 +62,17 @@ func (s *subsonicMediaProvider) IterateArtists(sortOrder string, filter mediapro
 			},
 			filter,
 		)
+	case ArtistSortRandom:
+		return s.baseArtistIterFromSimpleSortOrder(
+			func(artists []*subsonic.ArtistID3) []*subsonic.ArtistID3 {
+				newArtists := make([]*subsonic.ArtistID3, len(artists))
+				copy(newArtists, artists)
+				r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Minor suggestion: Since Go 1.20 the default random source is seeded automatically, so you can delete this line and just do `rand.Shuffle(...)` in place of `r.Shuffle`

Reply:
Actually I'll just make this quick fix myself. Works great on Jellyfin too, thanks!

Reply:
Good catch, thanks for pushing the fix!